pallets/fruniques/src/lib.rs

```diff
@@ -48,12 +46,16 @@ pub mod pallet {
 	pub enum Error<T> {
 		NoneValue,
 		NoPermission,
+		NotAdmin,
 		StorageOverflow,
 		NotYetImplemented,
 		// Too many fruniques were minted
 		FruniqueCntOverflow,
 		// The asset_id is not linked to a frunique or it doesn't exists
 		NotAFrunique,
+		KeyTooLong,
```

Comment:
Redacting an error description is a good practice, can you describe only the errors that you've declared? Describing the others will be specified on another github issue